zerver/tests/test_decorators.py

```diff
@@ -463,7 +463,7 @@ def test_login_deactivated_realm(self):
         """
         do_deactivate_realm(get_realm("zulip.com"))
         result = self.login_with_return("hamlet@zulip.com")
-        self.assertIn("has been deactivated", result.content.replace("\n", " "))
+        self.assertIn("has been deactivated", result.content.decode('utf-8').replace("\n", " "))
```

Comment:
It might be worth fixing the line-wrapping in `templates/zerver/deactivated.html` and then using `assert_in_response` here instead.  Probably fine to add that as a new commit on top rather than rebasing, though.